substrate/client/transaction-pool/src/fork_aware_txpool/dropped_watcher.rs

```diff
@@ -192,26 +289,30 @@ where
 							Command::RemoveView(key) => {
 								trace!(target: LOG_TARGET,"dropped_watcher: Command::RemoveView {key:?} views:{:?}",ctx.stream_map.keys().collect::<Vec<_>>());
 								ctx.stream_map.remove(&key);
-								ctx.transaction_states.iter_mut().for_each(|(_,state)| {
-									state.remove(&key);
+								ctx.ready_transaction_views.iter_mut().for_each(|(tx_hash,views)| {
+									trace!(target: LOG_TARGET,"[{:?}] dropped_watcher: Command::RemoveView ready views: {:?}",tx_hash, views);
```

Comment:
nit: Maybe break this on multiple lines, s/'{:?}",tx_hash'/'{:?}" ,tx_hash`

Reply:
:see_no_evil:

Reply:
done in: 7283471